tests/tests_integration/test_api/test_simulators/test_simulators.py

```diff
@@ -0,0 +1,290 @@
+import random
+import time
+import uuid
+
+import pytest
+
+from cognite.client import CogniteClient
+from cognite.client.data_classes.files import FileMetadata
+from cognite.client.data_classes.simulators.filters import (
+    SimulatorIntegrationFilter,
+    SimulatorModelRevisionsFilter,
+    SimulatorModelsFilter,
+    SimulatorRoutineRevisionsFilter,
+    SimulatorRoutinesFilter,
+)
+from cognite.client.data_classes.simulators.simulators import CreatedTimeSort
+from cognite.client.exceptions import CogniteAPIError
+from tests.tests_integration.test_api.test_simulators.seed.data import (
+    data_set_id,
+    resource_names,
+    simulator,
+    simulator_integration,
+    simulator_model,
+    simulator_model_revision,
+    simulator_routine,
+    simulator_routine_revision,
+)
+
+
+def truncated_uuid4():
+    return str(uuid.uuid4())[:8]
+
+
+model_unique_external_id = f"{resource_names['simulator_model_external_id']}_{truncated_uuid4()}"
+model_revision_unique_external_id = f"{resource_names['simulator_model_revision_external_id']}_{truncated_uuid4()}"
+simulator_routine_unique_external_id = f"{resource_names['simulator_routine_external_id']}_{truncated_uuid4()}"
```

Comment:
This won't work with how we run tests (xdist package). These need to be put behind a fixture so that dependent tests agree on what the random variable is.

Also, you can just import `random_string` from utils?